src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Modifier as PopperModifier, Options } from '@popperjs/core';
+import classnames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import messages from '../messages';
+import PopupBase from '../PopupBase';
+
+import './PopupList.scss';
+
+export type Modifier = PopperModifier<string, Partial<Options>>;
+
+export type Props = {
+    className?: string;
+    reference: HTMLElement;
```

Comment:
Let's update Popper.js to export the following type alias to match the popper.js types, then re-export it from PopupBase: `export type PopupReference = Element | VirtualElement;`, the latter of which is exported from popper.js.

We can then update the other popup components to import and use the new alias from PopupBase. It will also allow us to avoid some of the casting we're doing.